examples/tfengine/generated/org_foundation/audit/outputs.tf

```diff
@@ -1,17 +1,3 @@
-# Copyright 2020 Google LLC
```

Comment:
This lost the license header, we should put it back.

Reply:
Same in some other files lower down.

Reply:
An unfortunate result of merging deployments together. I will use xin's helper to add the headers instead of trying to do it manually ourselves in our templates to fix this later.